jme3-core/src/main/java/com/jme3/app/AppTask.java

```diff
@@ -100,7 +106,8 @@ public V get() throws InterruptedException, ExecutionException {
     }
 
     @Override
-    public V get(long timeout, TimeUnit unit) throws InterruptedException, ExecutionException, TimeoutException {
+    public V get(long timeout, TimeUnit unit) throws InterruptedException, ExecutionException, 
```

Comment:
"The prime directive of line-wrapping is: prefer to break at a higher syntactic level."  (Google Java Style Guide 4.5.1)

Here I recommend breaking the line before the `throws` keyword, so that the exceptions all appear on a single line.